src/event-stream/event-server.ts

```diff
@@ -190,6 +207,44 @@ async function handleClientMessage(msg: CoreNodeBlockMessage, db: DataStore): Pr
           value: hexToBuffer(event.contract_event.raw_value),
         };
         dbTx.contractLogEvents.push(entry);
+        if (
+          event.contract_event.topic === printTopic &&
+          event.contract_event.contract_identifier === bnsContractIdentifier
+        ) {
+          const functionName = getFunctionName(event.txid, parsedMsg.parsed_transactions);
+          if (nameFunctions.includes(functionName)) {
+            const attachment = parseNameRawValue(event.contract_event.raw_value);
+            const attachmentValue = await parseContentHash(attachment.attachment.hash);
```

Comment:
Nit: this function should be named something like `fetchAttachmentContent`

Reply:
@lgalabru can you weigh in on the safety here? Performing a fetch here results in the following:
* Core node POSTs event with bns update to sidecar, waits for HTTP 200 response.
* Sidecar sees this bns data, then performs this GET `/v2/attachments/<hash>`  back to the core node server.
* Core node replies to the attachment request, sidecar finishes and responds to the initial event POST request.

So in a roundabout way, the core node ends up waiting on it's own http request and server response. Could this cause any race conditions?

Reply:
Yeah that feels weird. The attachment could be embedded in the json instead.

Reply:
@zone117x 
Changed the function name here for now.